### DISCUSSION TITLE
Should nerd fonts v3 be working now ?

### DISCUSSION THREAD

**Discussion by nicos68** [jarun/nnn#1669] [Q&A] [answered]
I cloned the repo on my machine, checked master out, compiled using the command `sudo make strip install O_ICONS=1`, however the icons are still broken. I tried both jetbrains mono and fira code in alacritty. In neovim the icons were broken before I installed the V3 of the font and are fixed since I did. Should this be working or did I miss something?

**Comment by luukvbaal** [comment, marked as answer]
You need `O_NERD=1` for nerd font icons.

**Reply by nicos68** [reply]
oh, yeah, sorry :/
